### PR TITLE
Persist

### PR DESCRIPTION
**Pull request by thaistcosta**
Final one !!

Implemented redux-persist and added arisa's slices.

Ready to merge xuxus !!

Besitos 🍪 🦞

### CODE REVIEW

File: client/pages/_app.tsx [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+
+// let persistor = persistStore(store)
 
 function MyApp({ Component, pageProps }: AppProps) {
   return (
-    <>
-      <Component {...pageProps} />;
-    </>
+    <Provider store={store}>
+      <PersistGate loading={null} persistor={persistor}>
```
Comment (alexandra-plana) [inline client/pages/_app.tsx:15]: niceeeeeeeeee

**Review by alexandra-plana — APPROVED** [review]
this was beautiful!! ♥️🍅

File: client/common/store/index.store.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import {
+    persistStore,
+    persistReducer,
+    FLUSH,
+    REHYDRATE,
+    PAUSE,
+    PERSIST,
+    PURGE,
+    REGISTER,
+  } from 'redux-persist'
```
Comment (lexarisa) [inline client/common/store/index.store.ts:12]: LIFE SAVER!!!!

**Review by lexarisa — APPROVED** [review]
TTTTTTT i want to cry... what would i do without you guys <3 <3

**Pull request merged**